modules/i3s/src/lib/utils/url-utils.ts

```diff
@@ -6,13 +6,18 @@ import {Node3DIndexDocument, SceneLayer3D} from '../../types';
  * @returns url without search params
  */
 export function getUrlWithoutParams(url: string): string {
-  let urlWithoutParams;
+  let urlWithoutParams: string | null;
 
   try {
     const urlObj = new URL(url);
     urlWithoutParams = `${urlObj.origin}${urlObj.pathname}`;
+
+    // On Windows `new URL(url)` makes `C:\...` -> `null\...`
+    if (urlWithoutParams.startsWith('null')) {
```

Comment:
Nit: Allowing the string to become malformed and then fixing it up seems a bit sloppy. I would prefer that we checked before we formed the string (in this case I assume we could check `urlObj.origin`?)